mmeval/metrics/ms_ssim.py

```diff
@@ -0,0 +1,284 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+from typing import TYPE_CHECKING, Dict, List, Sequence, Tuple
+
+from mmeval.core import BaseMetric
+from mmeval.utils import try_import
+from .utils.image_transforms import reorder_image
+
+if TYPE_CHECKING:
+    from scipy import signal
+else:
+    signal = try_import('scipy.signal')
```

Comment:
`try_import` will return a `None` if the package is not installed. You should check for None and raise an error in `__init__`
Besides, we are making `scipy` a runtime dependency in PR #109 You can rebase to main after it has been merged.